package.json

```diff
@@ -59,14 +68,18 @@
     "react": ">=0.13"
   },
   "dependencies": {
+    "babel": "^5.6.23",
     "babelify": "^6.1.2",
     "browserify": "^10.2.4",
-    "es6-promise": "^2.3.0",
+    "bunyan": "^1.4.0",
     "fs-extra": "^0.20.1",
     "glob": "^5.0.10",
     "minimist": "^1.1.1",
     "mustache": "^2.1.2",
-    "object-assign": "^3.0.0"
+    "object-assign": "^3.0.0",
+    "react-docgen": "^1.3.0",
+    "react-simpletabs": "matthewgertner/react-simpletabs",
```

Comment:
This branch of `react-simpletabs` fixes https://github.com/pedronauck/react-simpletabs/issues/26, which is waiting for a PR review